src/main/java/org/owasp/esapi/logging/appender/ServerInfoSupplier.java

```diff
@@ -57,10 +58,19 @@ public String get() {
                 appInfo.append(request.getLocalAddr()).append(":").append(request.getLocalPort());
             }
         }
-        if (logAppName) {
-            appInfo.append("/").append(applicationName);
+
+        if (this.logAppName) {
+            if (this.applicationName != null && !this.applicationName.isEmpty()) {
```

Comment:
@jeremiahjstacey and @xeno6696 - Seeing this common boilerplate reminds me to let's make a note to use Lombok annotations in ESAPI 3.x. I'd do it now, but I really don't want to add another dependency at this point until we can break things up into smaller components and reduce the total dependencies where you are just using small pieces.